### PR TITLE
journalctl: emphasize command line argument in example descriptions

### PR DESCRIPTION
**Pull request by pkos98**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]
Goodly change!

File: pages/linux/journalctl.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
+- Show all messages from last [b]oot:
 
 `journalctl -b -1`
 
 - Show all messages with priority level 3 (errors) from this boot:
 
 `journalctl -b --priority={{3}}`
 
-- Follow new messages (like `tail -f` for traditional syslog):
+- [F]ollow new messages (like `tail -f` for traditional syslog):
```
Comment (navarroaxel) [inline pages/linux/journalctl.md:17]: This should keep the same case than the `-f` argument:
```suggestion
- [f]ollow new messages (like `tail -f` for traditional syslog):
```

File: pages/linux/journalctl.md [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
+- Show all messages from this [b]oot:
 
 `journalctl -b`
 
-- Show all messages from last boot:
+- Show all messages from last [b]oot:
 
 `journalctl -b -1`
 
 - Show all messages with priority level 3 (errors) from this boot:
```
Comment (navarroaxel) [inline pages/linux/journalctl.md:13]: Why not here?
```suggestion
- Show all messages with priority level 3 (errors) from this [b]oot:
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @pkos98

**Pull request merged**